packages/react-components/src/components/time-sync/rangeValidator.tsx

```diff
@@ -0,0 +1,31 @@
+import { DateRangePickerProps } from '@cloudscape-design/components';
+
+export const rangeValidator =
+  ({
+    dateRangeIncompleteError,
+    dateRangeInvalidError,
+  }: {
+    dateRangeIncompleteError: string;
+    dateRangeInvalidError: string;
+  }): DateRangePickerProps.ValidationFunction =>
+  (range: DateRangePickerProps.Value | null) => {
+    if (range?.type === 'absolute') {
+      const [startDateWithoutTime] = range.startDate.split('T');
+      const [endDateWithoutTime] = range.endDate.split('T');
+      if (!startDateWithoutTime || !endDateWithoutTime) {
+        return {
+          valid: false,
+          errorMessage: dateRangeIncompleteError,
+        };
+      }
+      const startTime = new Date(range.startDate).getTime();
+      const endTime = new Date(range.endDate).getTime();
+      if (startTime - endTime > 0) {
+        return {
+          valid: false,
+          errorMessage: dateRangeInvalidError,
+        };
+      }
+    }
+    return { valid: true, errorMessage: '' };
```

Comment:
curious .. why always return a empty errorMessage

Reply:
ahh this was a remnant of me testing something. will remove